lib/tasks/eu_exit_business_finder.rake

```diff
@@ -0,0 +1,108 @@
+  # facet titles from the JSON using their content IDs.
+  def title_from_facets(links)
+    business_finder_json = GdsApi.publishing_api_v2.get_content('42ce66de-04f3-4192-bf31-8394538e0734')
+    all_facets = business_finder_json['links']['facet_group'][0]['links']['facets']
+    all_facet_values = all_facets.map { |facet|
+      facet['links']['facet_values'].map { |facet_value|
+        title = facet_value['title']
+        replacement = EuExitFacetMigrationConfig::facet_value_labels_to_replace[facet_value['content_id']]
+        title = replacement if replacement
+        [facet_value['content_id'], title]
+      }.to_h
+    }.reduce({}, :merge)
+    matching_facet_values = all_facet_values.filter { |key, _title| links.include? key }
+    facet_titles = matching_facet_values.map { |_key, title| title }
+    comma_separated_facet_choices = facet_titles.map { |choice_name| "'#{choice_name}'" }.join(", ")
+    "EU Exit guidance for your business in the following #{'category'.pluralize(facet_titles.count)}: #{comma_separated_facet_choices}"
+  end
+end
+
+module EuExitFacetMigrationConfig
+  def self.facet_values_to_replace
+    [
+      {
+        "old_facet_value": "9f3476e1-8ff0-455d-a14e-003236b2797c",
+        "new_facet_value": "53f9ce4c-7cbb-447f-bdf1-a9b022896d3a"
+      },
+      {
+        "old_facet_value": "b4e507df-f067-4749-9468-3de120775216",
+        "new_facet_value": "24fd50fa-6619-46ca-96cd-8ce90fa076ce"
+      },
+      {
+        "old_facet_value": "356f46a0-17d3-4ba4-8952-8c02244f9045",
+        "new_facet_value": "18c3892e-8a0e-4884-906e-5938380eceee"
+      },
+      # "34a6edd0-46ea-4a76-ae80-8c96709d4f5" is worth noting as
+      # it's the result of a merge between two facets. The
+      # find-and-replace works the same way regardless.
+      {
+        "old_facet_value": "1e3e8abd-135d-4844-afa8-5c51df3d3c57",
+        "new_facet_value": "34a6edd0-46ea-4a76-ae80-8c96709d4f59"
+      },
+      {
+        "old_facet_value": "c1d8057c-76bf-431c-9ac8-6281a4b7b9ca",
+        "new_facet_value": "34a6edd0-46ea-4a76-ae80-8c96709d4f59"
+      },
+      # "94b3cfe2-af89-4744-b8d7-7fc79edcbc85" is a special case
+      # as it's been split into two facets. We should assume the
+      # user wants to remain subscribed to both.
+      {
+        "old_facet_value": "94b3cfe2-af89-4744-b8d7-7fc79edcbc85",
+        "new_facet_value": "9d54c591-f5ca-4d0c-a484-12d5591987cb"
+      },
+      {
+        "old_facet_value": "94b3cfe2-af89-4744-b8d7-7fc79edcbc85",
+        "new_facet_value": "afd45eef-743a-417d-9245-3eab8322116d"
+      }
+    ]
+  end
+
+  def self.facet_values_to_remove
+    [
+      "7536c0c4-fb41-43f4-a2c4-08f4fa9f5427",
+      "5faa1741-fc55-4110-b342-de92f6324118",
+      "14cf2a68-3297-44d3-ba01-a4426845b1b8",
+      "040649fc-4e2c-4028-b846-77fe3eebd1f7",
+      "94b3cfe2-af89-4744-b8d7-7fc79edcbc85"
+    ]
+  end
+
+  def self.facet_value_labels_to_replace
```

Comment:
I think this method title is unclear. I think a better name would be `facet_value_label_overrides`